data/formatters/generic.yaml

```diff
@@ -96,6 +96,16 @@ short_message:
 - 'Layer ID: {layer_id}'
 ---
 type: 'basic'
+data_type: 'dropbox:sync_history:entry'
+message:
+- 'File: {local_path}'
+- 'ID: {file_id}'
```

Comment:
ID => identifier

Reply:
Would "File ID" be acceptable?  I note further above in the same file there is "Layer ID" and "Container ID".